docs/developer-notes.md

```diff
@@ -76,6 +76,49 @@ Scuri uses [Jasmine's spy feature](https://jasmine.github.io/2.0/introduction.ht
 We loosely follow the [Jasmine Style Guide](https://github.com/CareMessagePlatform/jasmine-styleguide) when writing tests. 
 One main guideline is that a `describe` block should be created for each method / scenario under test, and an `it` block should be created for each property being verified.
 
+### E2E Testing
+
+#### CATcher Test Prep
```

Comment:
```suggestion
#### How the E2E tests work
```

Let's change this heading to `How the E2E tests work` - it's easier to understand for new developers?

Reply:
Yes thats better 👍